packages/theia-extension/yarn.lock

```diff
@@ -2,3 +2,189 @@
 # yarn lockfile v1
```

Comment:
remove

Reply:
resolved in 03c659a4b443a8c48d6aeaddcf6f30e1de3a948d